andela/rainier-cfh
# #151168131 send invites other users

#### What does this PR do?
Users are able to search for other users and send them game invite via Email.
#### Description of Task to be completed?
- When a user starts a game and wants to invite other friends to join the game
- User can click on the 'invite other users' button to see a modal with a search box
- User can then search and send invites to other users
- Invited users can see link to join the game in their emails
#### How should this be manually tested?
- On the main gaming page users should be able search and send invites to friends
#### Any background context you want to provide?
Used nodemailer to send emails. Nodemailer is an effective node package for sending in-app notifications in node application 
#### What are the relevant pivotal tracker stories?
Story type: Feature
Story Id: 151168131
Story: Users should be able to send invites to at most 11 people other than themselves
#### Screenshots 
![screenshot1](https://user-images.githubusercontent.com/24405172/30885156-27a0914c-a30a-11e7-9978-8893dee51f02.jpeg)
![screenshots 2](https://user-images.githubusercontent.com/24405172/30885396-001acb6e-a30b-11e7-9df0-fbfeaca74fdf.jpeg)
![screenshots 3](https://user-images.githubusercontent.com/24405172/30885410-0f7a4f30-a30b-11e7-945f-0a5152f02298.jpeg)

#### Questions:
